modules/custom/openy_gtranslate/templates/openy-gtranslate.html.twig

```diff
@@ -1,14 +1,20 @@
 {#
 /**
  * @file
- * Template for the Open Y Google Translate block
+ * Template for the OpenY Google Translate block
  */
 #}
 
-<div class="openy-google-translate" id="google_translate_element"></div>
+<div class="openy-google-translate hidden-xs hidden-sm" id="google_translate_element"></div>
 <script type="text/javascript">
-  function googleTranslateElementInit() {
-    new google.translate.TranslateElement({pageLanguage: drupalSettings.path.currentLanguage, layout: google.translate.TranslateElement.InlineLayout.SIMPLE}, 'google_translate_element');
-  }
+    function googleTranslateElementInit() {
+        new google.translate.TranslateElement(
+            {pageLanguage: drupalSettings.path.currentLanguage, layout: google.translate.TranslateElement.InlineLayout.VERTICAL},
+            'google_translate_element');
+        if(typeof(document.querySelector) == 'function') {
+            document.querySelector('.goog-logo-link').setAttribute('style', 'display: none');
+            document.querySelector('.goog-te-gadget').setAttribute('style', 'font-size: 0');
+        }
+    }
 </script>
-<script async type="text/javascript" src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
+<script type="text/javascript" src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
```

Comment:
let's keep async for PageSpeed optimization

Reply:
@podarok, fixed.